### PR TITLE
Feature/1.0.2

### PR DESCRIPTION
**Pull request by jackmarketon**
Few general bugfixes and updates

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "loader-utils": "~0.2.10",
     "node-sass": "^3.3.3",
     "object-assign": "~3.0.0",
+    "text-table": "^0.2.0",
```
Comment (dmachat) [inline package.json:31]: what's the extra dependency for?

Comment (jackmarketon) [inline package.json:31]: See: https://github.com/alleyinteractive/sasslint-webpack-plugin/issues/3

Appears the formatter file needs it, and somehow it didn't popup as extraneous for me. I think I missed it somehow originally (or it gets installed globally or something for some people).

**Comment by jackmarketon** [comment]
Going to plan on getting tests for these new features before officially merging.

**Comment by pebie** [comment]
:+1:

**Comment by dmachat** [comment]
lgtm :pineapple:

**Pull request merged**